_write.js

```diff
@@ -57,6 +51,16 @@ module.exports = function _write(httpMethod, options, callback) {
       catch (e) {
         err = e
       }
+
+      if (!ok) {
+        err = Error(httpMethod + ' failed with: ' + res.statusCode)
+        err.raw = res
+        err.body = result
+        callback(err)
+        res.resume()
```

Comment:
think we can remove this because `end` was called 

the Node docs:

> If no 'response' handler is added, then the response will be entirely discarded. However, if you add a 'response' event handler, then you must consume the data from the response object, either by calling response.read() whenever there is a 'readable' event, or by adding a 'data' handler, or by calling the .resume() method. Until the data is consumed, the 'end' event will not fire. Also, until the data is read it will consume memory that can eventually lead to a 'process out of memory' error.

Reply:
Yep, good call.